### PR TITLE
car docs: test print_docs_diff

### PR DESCRIPTION
**Pull request by maxime-desroches**
https://github.com/commaai/openpilot/issues/28267

### CODE REVIEW

File: selfdrive/car/tests/test_docs.py [hunk 1]
```diff
@@ -22,6 +27,13 @@ def test_generator(self):
     self.assertEqual(generated_cars_md, current_cars_md,
                      "Run selfdrive/car/docs.py to update the compatibility documentation")
 
+  def test_docs_diff(self):
+    with redirect_stdout(None):
```
Comment (sshane) [inline selfdrive/car/tests/test_docs.py:31]: is this to just get rid of `Dumping car info to...`?

File: selfdrive/car/tests/test_docs.py [hunk 2]
```diff
@@ -22,6 +27,13 @@ def test_generator(self):
...
+      dump_path = os.path.join(BASEDIR, "selfdrive", "car", "tests", "cars_dump")
```
Comment (sshane) [inline selfdrive/car/tests/test_docs.py:32]:
```suggestion
      dump_path = os.path.join(BASEDIR, "selfdrive/car/tests/cars_dump")
```

File: selfdrive/car/tests/test_docs.py (hunk 1 above)
Comment (maxime-desroches) [reply selfdrive/car/tests/test_docs.py:31]: Also to get rid of the potential output of print_car_info_diff. If we don't mind the output, I can remove the line

Comment (sshane) [reply selfdrive/car/tests/test_docs.py:31]: the diff should always be none, so should be fine

**Pull request merged**